ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/PutJobManagement_Test.java

```diff
@@ -536,4 +539,190 @@ public void getCompletedJobs() throws IOException, SignatureException, XmlProces
 
         assertThat(getCompletedJobsResponse.getStatusCode(), is(200));
     }
+
+    @Test
+    public void clearCompletedJobs() throws IOException, SignatureException, XmlProcessingException {
+        final ClearAllCompletedJobsSpectraS3Response clearAllCompletedJobsResponse = client
+                .clearAllCompletedJobsSpectraS3(new ClearAllCompletedJobsSpectraS3Request());
+
+        assertThat(clearAllCompletedJobsResponse.getStatusCode(), is(204));
+    }
+
+    @Test
+    public void putJobCreatedNotification() throws IOException, SignatureException {
+        final PutJobCreatedNotificationRegistrationSpectraS3Response response = client
+                .putJobCreatedNotificationRegistrationSpectraS3(new PutJobCreatedNotificationRegistrationSpectraS3Request("test@test.test"));
+
+        assertThat(response.getStatusCode(), is(201));
+    }
+
+    @Test
+    public void getJobCreatedNotification() throws IOException, SignatureException {
+        final UUID notificationUUID = client.putJobCreatedNotificationRegistrationSpectraS3(
+                new PutJobCreatedNotificationRegistrationSpectraS3Request("test@test.test"))
+                .getJobCreatedNotificationRegistrationResult().getId();
+
+        final GetJobCreatedNotificationRegistrationSpectraS3Response response = client
```

Comment:
In general I think its good practice to delete things that tests create to avoid the potential of conflict with other tests.